airflow/dag/base_dag.py

```diff
@@ -17,6 +17,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
+"""Base classes for DAG and DagBag."""
```

Comment:
Let's use 1 patter for all module docs eg.
"""
Base classes for DAG and DagBag.
"""

Reply:
@andriisoldatenko -> do you mean just changing it to be three-liner instead of one ? It does look neater so @BasPH maybe you can change it here before merging.

Reply:
yes

Reply:
Would you be okay with keeping the one-liner if it fits on a single line?

So
```python
"""Base classes for DAG and DagBag."""
```

If line length <= 110, otherwise:
```python
"""
Docs here... line 1
line 2
and more
"""
```

Reply:
I am good with that. PEP is always right.